src/Pact/Compile.hs

```diff
@@ -89,7 +90,7 @@ syntaxError' e s = mkInfo e >>= \i -> syntaxError i s
 doUse :: [Exp] -> Info -> Compile (Term Name)
 doUse as i = case as of
   [m] -> mkM m Nothing
-  [m,ELiteral (LString h) _] -> mkH h >>= mkM m
+  [m,eh@(ELiteral (LString h) _)] -> mkHash "use" h eh >>= mkM m . Just
```

Comment:
Nit: I think this would read better as `mkM m . Just =<< mkHash "use" h eh` so it doesn't switch direction.

Reply:
I like it. I had to try it because sometimes precedence doesn't work out right with `=<<` (maybe with dollar?) but here it's fine.